changehc/tests/test_sensor.py

```diff
@@ -53,12 +53,14 @@ def test_backfill(self):
     def test_fit_fips(self):
         date_range = pd.date_range("2020-05-01", "2020-05-20")
         all_fips = self.combined_data.index.get_level_values('fips').unique()
-        sample_fips = nr.choice(all_fips, 10)
+        sample_fips = all_fips[:50]
```

Comment:
If we ignore fips that have all nan for rate, is this line necessary?

Reply:
Probably not necessary, but non-deterministic tests just seem like a bad idea to me in general.